ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java

```diff
@@ -352,4 +365,14 @@ public void waitForIntegrationPodShutdown(String integartionPodName) throws Inte
                 OpenShiftWaitUtils.areNoPodsPresent(integartionPodName),1000,5 * 60 *1000);
     }
 
+    @And("^Camilla drags exported integration \"([^\"]*)\" file to drag and drop area$")
+    public void importIntegrationViaDragAndDrop(String integrationName) throws InterruptedException {
+        Assertions.assertThat(listComponent.importIntegrationViaDragAndDrop(integrationName)).isEqualTo(true);
```

Comment:
This can be simplified to?
 `Assertions.assertThat(listComponent.importIntegrationViaDragAndDrop(integrationName)).isTrue();`